Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/config/PortalSecurityConfig.java

```diff
@@ -34,6 +34,21 @@
 @ConditionalOnProperty(prefix = "keycloak", name = "enabled", havingValue = "false")
 public class PortalSecurityConfig extends WebSecurityConfigurerAdapter {
 
+  private static final String[] SWAGGER2_AUTH_WHITELIST = {
```

Comment:
Seems like PortalSecurityConfig and PortalKeyCloakSecurity both share the same whitelists? Would be worth adding to a common whitelist constants, or define in an externalised application configuration.